css/quote-generator.css

```diff
@@ -0,0 +1,161 @@
+.page {
+  width: 100vw;
+}
+
+.top-buttons {
+  margin: 5vh 10vw 15vh 10vw;
+  display: flex;
+  justify-content: space-between;
+}
+
+.top-button {
+  font-family: "Raleway";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 24px;
+  border: none;
+  color: #4f4f4f;
+  background-color: #ffffff;
+  box-shadow: 0 4px 2px -2px gray;
+  cursor: pointer;
+}
+
+.active {
+  border-bottom: solid 2px blue;
+}
+
+.top-icon {
+  vertical-align: bottom;
+  font-size: 24px !important;
+}
+
+.grid-container {
+  max-width: 35vw;
+  margin: auto;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  gap: 10vh;
+}
+
+.quote {
+  font-family: "Raleway";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 36px;
+  flex-basis: 100%;
+  padding-left: 5vw;
+  border-left: solid 0.5vw #f7df94;
+}
+
+.quote-author-container {
+  display: flex;
+  flex-flow: row wrap;
+  align-content: center;
+  margin-left: 5.5vw;
+}
+
+.quote-author {
+  font-family: "Raleway";
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
```

Comment:
Use css variables for your typography styles to reuse them

Something like this: `--font-size-default: 24px`

Then use it: `var(--font-size-default)

https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties